### PR TITLE
new endpoint for getting the full address by postal code and house number

### LINKED ISSUE #3: address check lambda

### PR DESCRIPTION
**Pull request by petarblazevski**
Closes #3

### CODE REVIEW

File: address/get.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+import { failure, success } from '../libs/response-lib';
+
+export const main = async event => {
+  const { postalCode, houseNumber } = event.pathParameters;
```
Comment (IAfanasovMob) [inline address/get.js:9]: would be nice to add JSDocs with variable types

File: address/get.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        },
+        params: {
+          'subscription-key': mapsKey,
+          postalCode: postalCode.replace(' ', ''),
+          houseNumber
+        }
+      }
+    );
+
+    const data = response.data.results[0];
```
Comment (IAfanasovMob) [inline address/get.js:30]: especially here it is impossible to know the fields of `data` unless one do the manual test

**Review by IAfanasovMob — APPROVED** [review]
it would be really great to add JSDoc
overall really good!

File: address/get.js (hunk 2 above)
Comment (petarblazevski) [reply address/get.js:30]: I added some JSDoc, but not sure if it is the correct way. Never before have I used this approach honestly. If you see if this works. Webstorm doesn't provide me with any info whatsoever

Comment (IAfanasovMob) [reply address/get.js:30]: looks good, thanks!
VSCode supports it and does a clever suggestions

**Pull request merged**